### DISCUSSION TITLE
Module: Critical CVE Scanning

### DISCUSSION THREAD

**Discussion by aconite33** [blacklanternsecurity/bbot#726] [Ideas]
Expand on the Nuceli Module to target specific Critical CVE's to be scanned for

Examples:

* CVE-2017-0143 - EternalBlue
* CVE-2019-0708 - BlueKeep
* CVE-2021-34527 - Print NightMare
* CVE-2020-1472 - ZeroLogon
* CVE-2021-36942 - PetitPotam
* MS08-0167
* Log4j

**Comment by SpamFaux** [comment]
Passing over to jr devs for module creation.

**Comment by liquidsec** [comment]
we dont want to reinvent the wheel (the wheel is nuclei) for these
